packages/devtools/lib/src/timeline/cpu_call_tree.dart

```diff
@@ -59,17 +60,13 @@ class CpuCallTree extends CoreElement {
   }
 
   void show() async {
-    attribute('hidden', false);
+    hidden(false);
```

Comment:
nit: consider just making show a getter instead. Also remove async if it isn't needed.